src/main/java/no/nav/pto/veilarbportefolje/opensearch/OpensearchQueryBuilder.java

```diff
@@ -446,64 +447,65 @@ static List<BoolQueryBuilder> byggAktivitetFilterQuery(Filtervalg filtervalg, Bo
     static SearchSourceBuilder sorterQueryParametere(String sortOrder, String
             sortField, SearchSourceBuilder searchSourceBuilder, Filtervalg filtervalg, BrukerinnsynTilganger brukerinnsynTilganger) {
         SortOrder order = "ascending".equals(sortOrder) ? SortOrder.ASC : SortOrder.DESC;
+        Sorteringsfelt sorteringsfelt = Sorteringsfelt.nameFromValue(sortField);
```

Comment:
Relatert til ein annan kommentar eg hadde; det hadde vore digg om `sortField` allereie var av typen `Sorteringsfelt` når den kjem inn som argument til `sorterQueryParametere`. Dersom vi går for det så anbefalar eg at ein gjer valideringen (dvs. forsøk å parse `sortField` som eit `Sorteringsfelt`) så tidleg som mogleg, aller helst i høvesvis `EnhetController`/`VeilederController`. Sjå f.eks. linje 63-65 i `EnhetController` kor vi gjer annan validering.